cmd/minishift/cmd/addon/addon_disable.go

```diff
@@ -51,6 +52,10 @@ func runDisableAddon(cmd *cobra.Command, args []string) {
 		atexit.ExitWithMessage(0, fmt.Sprintf(noAddOnToDisableMessage, addOnName))
 	}
 
+	if !addOnManager.Get(addOnName).IsEnabled() {
+		atexit.ExitWithMessage(0, fmt.Sprintf(addOnAlreadyDisabledMessage, addOnName))
+	}
+
```

Comment:
@ulyssessouza Please add a unit test around this look into the `addon_disable_test.go` file.

Reply:
@anjannath To write an unit test for this we need have code to mock installation of an add-on. I am fine with not having a unit test for this as it is a not critical piece.